### DISCUSSION TITLE
Differentiate subdomains in the "Top pages" sections

### DISCUSSION THREAD

**Discussion by baptisteArno** [plausible/analytics#946] [Anything Else]
I allow my users to have a dedicated subdomain to my service: client1.example.com, client2.example.com, etc...

The problem is that if a user navigates to the root of one of these domains, I will not be able to know that it's from this particular subdomain in Plausible. https://example.com/, https://client1.example.com/, https://client2.example.com/ all count as the same page in the "Top pages" section.

Is there a way of avoiding this behavior?

**Comment by metmarkosaric** [comment]
hi @baptisteArno! You can add individual subdomains as their own sites within your Plausible account. that way each subdomain gets its own dashboard independent from the rest of your site.

**Reply by baptisteArno** [reply]
hey @metmarkosaric, thank you for the quick reply.
Unfortunately, that's not what I want as it would result in too many dashboards that I need to set up by hand each time a client creates a subdomain. 
Is there a way to silently tell plausible that a certain subdomain should be considered as a certain page? Example: Plausible detects `client1.example.com`, I want it to log a page view as `/client1`

**Reply by metmarkosaric** [reply]
that's not possible at the moment unfortunately. we're not far away from releasing our sites API which will allow you to add domains to Plausible automatically. you can follow the progress here https://github.com/plausible/analytics/discussions/150

**Reply by baptisteArno** [reply]
Ok, that would be a solution! And then, would `example.com` be able to ignore data from `client1.example.com` to avoid duplicated events throughout the dashboards?

**Reply by metmarkosaric** [reply]
as long as you don't install the script with data-domain attribute `example.com` on `client1.example.com` it won't be counting the stats from `client1.example.com`

**Reply by baptisteArno** [reply]
Sorry to sound persistent but I still feel it would be **a must** to have the subdomain as an attribute to consider for a page view.

```
https://example.com/
https://app.example.com/
https://admin.example.com/
```
I feel these shouldn't count as the same page view. Don't you think?

**Reply by metmarkosaric** [reply]
Plausible essentially sees different subdomains as different sites so they would only count as the same page view if you add them as one site in Plausible. We may be able to change this in the future to allow your use case but it is not on our short term roadmap. We are a small team with limited development resources so need to take it one step at a time with focus being on the most frequently requested features first

**Reply by baptisteArno** [reply]
Would a PR implementing this feature in a clean way would be accepted?

**Reply by metmarkosaric** [reply]
that would be welcomed for sure! the feature has been asked for before we just haven't got to doing in a clean way so it fits with our current dashboard and doesn't make the interface complicated for 95%+ of users who don't need this functionality. you can read our thoughts on how it can be done best here https://github.com/plausible/analytics/pull/478#issuecomment-805816732
